docs/resources/ims_obs_iso_image.md

```diff
@@ -0,0 +1,134 @@
+
+* `name` - (Required, String) Specifies the name of the image.
+  The valid length is limited from `1` to `128` characters.
+  The first and last letters of the name cannot be spaces.
+  The name can contain uppercase letters, lowercase letters, numbers, spaces, chinese, and special characters (-._).
+
+* `image_url` - (Required, String, ForceNew) Specifies the URL of the external image file in the OBS bucket, the format
+  is **OBS bucket name:image file name**. The storage category for OBS bucket and image file must be OBS standard
+  storage, and the image file must be ISO format. Changing this parameter will create a new resource.
+
+* `min_disk` - (Required, Int, ForceNew) Specifies the minimum size of the system disk, in GB unit.
+  Changing this parameter will create a new resource.
+  + When the operating system is Linux, the value ranges from `10` to `1,024`.
+  + When the operating system is Windows, the value ranges from `20` to `1,024`.
+
+* `os_version` - (Required, String, ForceNew) Specifies the operating system version of the image.
+  Changing this parameter will create a new resource.
+  For its values, see [API docs](https://support.huaweicloud.com/intl/en-us/api-ims/ims_03_0910.html).
+
+* `description` - (Optional, String) Specifies the description of the image.
+
+* `is_config` - (Optional, Bool, ForceNew) Specifies whether to automatically configure. The value can be **true** or
+  **false**. Defaults to **false**. Changing this parameter will create a new resource.
+  About the content of automatic backend configuration, please refer to
+  [API docs](https://support.huaweicloud.com/intl/en-us/ims_faq/ims_faq_0020.html).
+
+* `cmk_id` - (Optional, String, ForceNew) Specifies the custom key for creating encrypted image.
+  Changing this parameter will create a new resource.
+
+* `architecture` - (Optional, String, ForceNew) Specifies the schema type of the image. The value can be **x86** or
+  **arm**. Defaults to **x86**. Changing this parameter will create a new resource.
+
+* `max_ram` - (Optional, Int) Specifies the maximum memory of the image, in MB unit.
+
+* `min_ram` - (Optional, Int) Specifies the minimum memory of the image, in MB unit.
+
+* `tags` - (Optional, Map) Specifies the key/value pairs to associate with the image.
+
+* `enterprise_project_id` - (Optional, String) Specifies the enterprise project ID to which the IMS image belongs.
+  For enterprise users, if omitted, default enterprise project will be used.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The ID of the image.
+
+* `status` - The status of the image. The value can be **active**, **queued**, **saving**, **deleted**, or **killed*,
+  only image with a status of **active** can be used.
+
+* `visibility` - Whether the image is visible to other tenants.
+
+* `image_size` - The size of the image file, in bytes unit.
+
+* `os_type` - The operating system type of the image.
```

Comment:
Please list the enumeration values ​​supported by this field.

Reply:
fixed.